src/components/List.js

```diff
@@ -51,6 +54,48 @@ const List = ({ firestore }) => {
     setInputText('');
   };
 
+  //* Created two lists
+  const inactiveList = [];
+  const activeList = [];
+
+  //* Used this function to divide active and inactive items
+  filteredList.map(item => {
+    let formatedLastPurchaseDate = dayjs.unix(
+      item.lastPurchasedDate['seconds'],
+    );
+    let formatedToday = dayjs();
+    let difference = formatedToday.diff(formatedLastPurchaseDate, 'd');
+    if (
+      item.numberOfPurchases <= 1 ||
+      parseInt(difference) >= item.nextPurchase * 2
+    ) {
+      inactiveList.push(item)
+    } else {
+      activeList.push(item)
+    }
+  });
+
+
+ //* Used this function to sort both the lists
+  const lists = (arr) => {
```

Comment:
When I was skimming the code, I thought the lists function might have been a built in function and wasn't totally sure about what was happening when it was called below. Could perhaps rename the function to something like `sortLists` to make it more clear what is happening.